hummingbot/connector/exchange/bitfinex/bitfinex_exchange.pyx

```diff
@@ -956,8 +961,11 @@ cdef class BitfinexExchange(ExchangeBase):
             #   an order from the order book  - sell
             data["maker_order_id"] = content[3] if content[4] > 0 else None
             data["taker_order_id"] = content[3] if content[4] < 0 else None
+            data["trade_id"] = content[0]
             data["price"] = content[5]
             data["amount"] = content[4]
+            data["fee"] = content[9]
+            data["fee_currency"] = content[10]
```

Comment:
I am not sure, I doubt about this. What makes me doubt is that it seems some elements in the response can be absent (for example line 1039). I am not sure how that could happen, because the code that parses the response and create the new dictionary seems to add the price element.

Reply:
I suppose you're right. Other than readability, I dont see any drawbacks with using the code as it is.